bedrock/mozorg/redirects.py

```diff
@@ -0,0 +1,238 @@
+    redirect('^firefox/backtoschool/firstrun/?$', 'firefox.firstrun'),
+
+    # bug 824126, 837942
+    redirect('^ports/qtmozilla(?:/|/index.html)?$', 'https://wiki.mozilla.org/Qt'),
+    redirect('^ports/os2/?$', 'https://wiki.mozilla.org/Ports/os2'),
+    redirect('^ports(?P<path>.*)', 'http://www-archive.mozilla.org/ports{path}'),
+
+    redirect(r'^b2g', 'firefox.partners.index'),
+
+    # Bug 781914
+    redirect(r'^contribute/areas.html$', 'mozorg.contribute'),
+    redirect(r'^contribute/universityambassadors',
+             'mozorg.contribute.studentambassadors.landing'),
+
+    # Bug 1144949
+    redirect(r'^contribute/page/?$',
+             'https://wiki.mozilla.org/Webdev/GetInvolved/mozilla.org'),
+
+    # Bug 763665, 1148127
+    redirect(r'^projects/$', 'firefox.family.index'),
+
+
+    # Bug 792185 Brand Toolkit -> Style Guide
+    redirect(r'^firefox/brand/$', 'styleguide.home'),
+    redirect(r'^firefox/brand/platform/$',
+             'styleguide.identity.firefox-family-platform'),
+    redirect(r'^firefox/brand/identity/$',
+             'styleguide.identity.firefox-branding'),
+    redirect(r'^firefox/brand/identity/channel-logos/$',
+             'styleguide.identity.firefox-channels'),
+    redirect(r'^firefox/brand/identity/wordmarks/$',
+             'styleguide.identity.firefox-wordmarks'),
+    redirect(r'^firefox/brand/identity/typefaces/$',
+             'styleguide.communications.typefaces'),
+    redirect(r'^firefox/brand/artwork/$', 'styleguide.home'),
+    redirect(r'^firefox/brand/artwork/gear/$', 'styleguide.home'),
+    redirect(r'^firefox/brand/website/$',
+             'styleguide.websites.sandstone-intro'),
+    redirect(r'^firefox/brand/website/domain-strategy/$',
+             'styleguide.websites.domains-overview'),
+    redirect(r'^firefox/brand/copy/$', 'styleguide.communications.copy-tone'),
+    redirect(r'^firefox/brand/copy/l10n/$',
+             'styleguide.communications.translation'),
+    redirect(r'^firefox/brand/copy/rules/$',
+             'styleguide.communications.copy-rules'),
+    redirect(r'^firefox/brand/downloads/$', 'styleguide.home'),
+
+    # Bug 1186373
+    redirect(r'^firefox/hello/npssurvey/$',
+             'https://www.surveygizmo.com/s3/2227372/Firefox-Hello-Product-Survey',
+             permanent=False),
+
+    # Bug 1071318
+    redirect(r'^firefox/mobile/$', 'firefox.android.index'),
+
+    # Bug 804810 Identity Guidelines -> Style Guide
+    redirect(r'^foundation/identity-guidelines/index.html', 'styleguide.home'),
+    redirect(r'^foundation/identity-guidelines/mozilla-foundation.html',
+             'styleguide.identity.mozilla-branding'),
+    redirect(r'^foundation/identity-guidelines/thunderbird.html',
+             'styleguide.identity.thunderbird-logo'),
+
+    # Bug 945474 - delete Marketplace marketing product page
+    # and redirect
+    redirect(r'^apps/$', 'https://marketplace.firefox.com/'),
```

Comment:
The mcom-tests revealed that this only works if you include the trailing slash: `/apps/` redirects to marketplace, but `/apps` gets redirected to `/en-US/apps/` which is a 404.